reactapp/src/components/Header.tsx

```diff
@@ -0,0 +1,9 @@
+
+import React from 'react';
+
+export const Header: React.FC = () => (
+    <div>
+        <h2>qoutequack</h2>
```

Comment:
@jarle123 Vurder å bruke h1 som toppnivå header for semantisk riktighet 😀

Reply:
Good point, changing it!